src/components/Header/Header.tsx

```diff
@@ -31,9 +40,13 @@ const Header = (props: any): JSX.Element => {
       {!isMovieDetailsShown ? (
         <>
           <h1>Movie World </h1>
-          <MwButton onClickInternal={showModal} className="addEditDialogButton" buttonName={title}></MwButton>
+          <MwButton
+            onClickInternal={onAddMovieClick}
```

Comment:
No idea how I missed this the last time, but why `onClickInternal` and not just `onClick`?

Reply:
Last time you said to change from onClick to onClickInternal ))

Reply:
will change